### DISCUSSION TITLE
Is it possible to make the Authorization checks (Check APIs) case insensitive easily?

### DISCUSSION THREAD

**Discussion by velmohan** [openfga/community#186] [Q&A] [answered]
When it comes to Tuple values, the checks should actually be case insensitive and so we do not have to expect applications to remember the cases of the values. Or should we say, all Tuple values should be in lower case or vice versa? This applies mainly for User and Object values as the Relation is defined by Authorization Models and so everyone would know what cases the relation names are in.

**Comment by aaguiarz** [comment, marked as answer]
Hi @velmohan 

No, that's not possible. You can wrap our SDKs so you normalize the identifiers before calling the service.

**Reply by velmohan** [reply]
Thanks @aaguiarz. That was the route I was taking but wanted to check just in case there was any easier way.
